pkgs/os-specific/linux/power-profiles-daemon/default.nix

```diff
@@ -58,6 +52,13 @@ stdenv.mkDerivation rec {
     gobject-introspection
     wrapGAppsNoGuiHook
     python3.pkgs.wrapPython
+    (python3.pythonForBuild.withPackages (ps: with ps; [
+      pygobject3
```

Comment:
But, don't these belong to checkInputs?

Reply:
they're checked for during the configuring, added a comment